webapp/components/channel_select.jsx

```diff
@@ -77,3 +117,9 @@ export default class ChannelSelect extends React.Component {
         );
     }
 }
+
+ChannelSelect.defaultProps = {
```

Comment:
We use a get function for the propTypes but this for default props. We should be consistent.